paasta_tools/cli/cmds/spark_run.py

```diff
@@ -395,6 +395,14 @@ def add_subparser(subparsers):
         default=DEFAULT_AWS_REGION,
     )
 
+    aws_group.add_argument(
+        "--assume-pod-identity",
+        help="If running running in the context of a Kubernetes pod with pod identity set, "
```

Comment:
```suggestion
        help="If running in the context of a Kubernetes pod with pod identity set, "
```

Reply:
afaik: this is functionally a noop right now - paasta spark-run will be called outside the context of a k8s pod (only jupyter will start the driver inside k8s)